fastai/callbacks/mlflow.py

```diff
@@ -0,0 +1,44 @@
+"A `Callback` that saves tracked metrics and notebook file into MLflow server."
+from ..torch_core import *
+from ..callback import *
+from ..basic_train import Learner, LearnerCallback
+#This is an optional dependency in fastai.  Must install separately.
+try: import mlflow
+except: pass
+
+class MLFlowTracker(LearnerCallback):
+    "A `TrackerCallback` that tracks the loss and metrics into MLFlow"
+    def __init__(self, learn:Learner, exp_name: str, params: dict, nb_path: str, uri: str = "http://localhost:5000"):
+        super().__init__(learn)
+        self.learn = learn
```

Comment:
Please initiliaze all those attributes on the same line (at least as many as can fit):
```
self.learn,self.exp_name,... = learn,exp_name,...
```
(fastai style guide)